gdp-src-build/conf/templates/renesas-rcar-gen3.local.inc

```diff
@@ -0,0 +1,143 @@
+# Include generic GDP local configuration
+include local.inc
+
+# Add R-Car Gfx and MMP packages to install
+IMAGE_INSTALL_append_rcar-gen3 = "libdrm-kms kernel-module-gles gles-user-module libgbm-dev packagegroup-gstreamer1.0-plugins weston-bin kernel-module-pvrsrvkm"
+
+MACHINE ?= "m3ulcb"
+
+# This sets the SoC
+# H3: r8a7795, M3: r8a7796
+#SOC_FAMILY = "r8a7795"
+SOC_FAMILY ?= "r8a7796"
+
+# Add systemd configuration
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
```

Comment:
Firstly thanks for the review Changhyeok.

Although it is a duplication I think it would be best to keep it. My reasoning is that local.conf entries like this are pretty portable in describing how you might configure the Yocto BSP to work with a Genivi based distro. So someone might take this a starting point to use in their own platform and that may not be meta-ivi based.

It is also useful to minimise minor differences between platforms. So an R-Car Gen 3 local.conf for GDP may be consistent with one for a different distro.